Controller/ClientController.php

```diff
@@ -76,6 +76,8 @@ public function show(Request $request, Process $process, CprServiceInterface $cp
      */
     public function edit(Request $request, Process $process): Response
     {
+        $this->denyAccessUnlessGranted('edit', $process);
```

Comment:
Could/should this be replaced with an `@IsGranted("edit", subject="process")` (or `@Security`) annotation (https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/security.html)? `@IsGranted` is used in `ProcessController` and for consistency we should do it the same way whenever possible.

Reply:
The annotation couldn't fetch subject from class route annotation, so a call to the denyAccessUnlessGranted method was used instead. The annotations in the ProcessController is now replaced with calls to the method mentioned above.